tests/testutils/goroutineassistant.go

```diff
@@ -46,14 +51,38 @@ func (a *GoroutineAssistant) Done() {
 }
 
 func (a *GoroutineAssistant) Wait() {
-	done := make(chan struct{})
 	go func() {
 		a.wg.Wait()
-		done <- struct{}{}
+		a.s <- nil
 	}()
-	select {
-	case s := <-a.s:
-		a.t.Fatalf(s)
-	case <-done:
+
+	if err := <-a.s; err != nil {
```

Comment:
This is a kind of stop-gap solution - I'd prefer it to be a part of `rktRunCtx`. Maybe good enough at the moment.